Сишарпэээ/Program.cs

```diff
@@ -1,3 +1,67 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
-Console.ReadKey();
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Versioning;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exam_2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int i;
+            Console.WriteLine("Введите чётное число, которое больше либо равно 4");
+            i = int.Parse(Console.ReadLine());
+            if (i % 2 == 0)
+            {
+                var result = ShowPrimeSummands(i);
+                if (result != null)
```

Comment:
Вместо проверки на `null` проверка на непустой массив.
```suggestion
                if (result.Length > 0)
```